opl/cluster_read.py

```diff
@@ -88,18 +88,24 @@ def measure(self, start, end, name, monitoring_query, monitoring_step):
         # Check that what we got back seems OK
         json_response = response.json()
         logging.debug("Response: %s" % json_response)
-        assert json_response['status'] == 'success', \
-            "'status' needs to be 'success'"
-        assert 'data' in json_response, \
-            "'data' needs to be in response"
-        assert 'result' in json_response['data'], \
-            "'result' needs to be in response's 'data'"
-        assert len(json_response['data']['result']) != 0, \
-            "missing 'response' in response's 'data'"
-        assert len(json_response['data']['result']) == 1, \
-            "we need exactly one 'response' in response's 'data'"
-        assert 'values' in json_response['data']['result'][0], \
-            "we need expected form of response"
+
```

Comment:
Do not remove these asserts. You whould have to remove them in other plugins as well. Just replace that `traceback.print_exc()` on line ~312 with something that would log that traceback as a debug and print a warning there.